service/graph_utils.py

```diff
@@ -0,0 +1,74 @@
+import itertools
+
+import networkx as nx
+from networkx.drawing.nx_agraph import to_agraph
+
+
+class GraphUtils:
+
+    def handle_graph(self, vms_data, fw_rules_data):
+        nodes = [vm.vm_id for vm in vms_data]
+        edges = self.prepare_graph_edges(vms_data, fw_rules_data)
+        vms_graph = self.create_graph(nodes=nodes, edges=edges)
+        self.create_graph_visualization(vms_graph)
+        prepare_vm_potentials_attackers_dict = self.prepare_vm_potentials_attackers_dict(vms_graph)
+        return prepare_vm_potentials_attackers_dict
```

Comment:
why is this function returning a value? this should be the __init__ method.....!